### ISSUE TITLE
"Encrypt" and "Trust Server Certificate" tooltip is not accessible using keyboard: A11y_MySQL extension for Azure Data Studio_Connect to a MySQL Server_Keyboard

### ISSUE THREAD

**Issue by Swati1700** [microsoft/azuredatastudio-mysql#237]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”
**Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.**

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM_MySQL extension for Azure Data Studio_Win32_Mar2024; #MySQL extension for Azure Data Studio; #Win32; #Benchmark; #Win11; #WCAG2.1.1; #Keyboard; #FTP;

### Environment and OS details:
Azure Data Studio Version: 1.48.0 (user setup)
Windows 11 pro Version 23H2
OS build 22631.3296

### Repro Steps:

1. Launch Azure Data Studio.
2. Tab to the new connection button and press enter.
3. Tab to "Encrypt" and "Trust Server Certificate" and observe tooltip is not accessible using keyboard.

### Actual Result:
"Encrypt" and "Trust Server Certificate" tooltip is not accessible using keyboard.

### Expected Result:
"Encrypt" and "Trust Server Certificate" tooltip should be accessible using keyboard.

### User Impact:
Keyboard users and screen reader users will not get tooltip information if it is not accessible.

### Attachment:
[Keyboard.zip](https://github.com/microsoft/azuredatastudio-mysql/files/14722406/Keyboard.zip)

**Comment by rishky-msft** [comment]
The Encrypt and Trust Server Certificate are not present in MySQL Extension dialog.
This issue is related to Azure Data Studio SQL Server. Transferring it to microsoft/azuredatastudio repo
